src/styles/_catalog.scss

```diff
@@ -1,10 +1,20 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  justify-content: center;
   margin: 0 auto;
 
+  @media (min-width: 488px) {
+    grid-template-areas: "card card";
```

Comment:
Better to use template with `repeat` instead of areas.
Areas are used for complex grid structures